code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -626,7 +626,11 @@
 		O.equip(new_spawn, FALSE, new_spawn.client)
 		SSjob.equip_loadout(null, new_spawn, FALSE)
 		SSquirks.AssignQuirks(new_spawn, new_spawn.client, TRUE, TRUE, null, FALSE, new_spawn)
+		new_spawn.AddElement(/datum/element/ghost_role_eligibility)
+		ADD_TRAIT(new_spawn, TRAIT_SIXTHSENSE, GHOSTROLE_TRAIT)
```

Comment:
Isn't the "ghost cafe" more or less the same location of the "ninja prison"?
Shouldn't a few chat lines to warn against blurting out OOC in IC or deadchat while inside the body be needed?

Reply:
Yeah, I'll do that with the merge conflict fix